### DISCUSSION TITLE
Add pyserial install to Wiki ?

### DISCUSSION THREAD

**Discussion by Doreapp** [svpcom/wfb-ng#157] [Ideas]
While I was installing several times the repository, I noticed an error linked to `serial` python library.

It is easily solved using 
```
sudo pip install pyserial
```
But it could be added to wiki, to avoid needing to debug this error
(And the install of `pip` too, for brand-new Ubuntu images)

Something like
```
sudo apt-get install python3-pip
sudo pip install pyserial
```

**Comment by svpcom** [comment]
Added in f46db3758b02611ba849612f5a6693b3179668c1
